packages/prism-editor/src/PrismEditor.tsx

```diff
@@ -62,22 +63,16 @@ const LineContent = styled.span`
   display: table-cell;
 `
 
+/**
+ * The PrismEditor provides a useful implementation of prism-react-renderer
+ * for Looker apps. All @looker/sdk languages are supported.
+ * TODO: LookML syntax highlighting
+ */
 export const PrismEditor: FC<PrismEditorProps> = ({
-  language,
+  language = 'json',
   code,
-  pattern,
+  pattern = '',
 }) => {
```

Comment:
Can you add some tests for this component as well please? Let me know if you'd like to pair up :)